src/common/src/array/interval_array.rs

```diff
@@ -34,19 +34,16 @@ mod tests {
         }
         let ret_arr = array_builder.finish();
         for v in ret_arr.iter().flatten() {
-            assert_eq!(v.get_years(), 1);
             assert_eq!(v.get_months(), 12);
             assert_eq!(v.get_days(), 0);
         }
         let ret_arr = IntervalArray::from_iter([Some(IntervalUnit::from_ymd(1, 0, 0)), None]);
         let v = ret_arr.value_at(0).unwrap();
-        assert_eq!(v.get_years(), 1);
         assert_eq!(v.get_months(), 12);
         assert_eq!(v.get_days(), 0);
```

Comment:
As the test itself shows, `1 year 0 month 0 day` is returning
```
        assert_eq!(v.get_years(), 1);
        assert_eq!(v.get_months(), 12);
        assert_eq!(v.get_days(), 0);
```
which is inconsistent and confusing.